test/test-is-buffer-transferred.ts

```diff
@@ -8,8 +8,8 @@ function wait () {
 
 test('transferable objects must be transferred', async ({ equal }) => {
   const pool = new Piscina({
-    filename: resolve(__dirname, 'fixtures/send-buffer-then-get-length.js'),
-    useAtomics: false
+    filename: resolve(__dirname, 'fixtures/send-buffer-then-get-length.js')
+    // useAtomics: false // let's see if turning on atomics makes a difference
```

Comment:
This is the culprit? 🤔

Reply:
oh _wow_, I stopped paying attention to this over the weekend and kept getting failures in act, so didn't notice tests pass.

There's no way this specific line change was the culprit, since more than just this test failed last run. Since tests pass anyways I have no hypothesis anymore. Have you observed your CI tests flake like this in the past?

Reply:
Not really, we had some issues when migrating to v20, but I don't think that should be the culprit then 🤔 

Can you uncomment it and see if that solves the issue?

Reply:
sure, done. let's see!

Reply:
well, i'll take it!